docs/reference/aggregations/pipeline/bucket-correlation-aggregation.asciidoc

```diff
@@ -31,21 +31,29 @@ The correlation function to execute.
 The configuration to calculate a count correlation. This function is designed for
 determining the correlation of a term value and a given metric. Consequently, it
 needs to meet the following requirements.
++
+--
 
 * The `buckets_path` must point to a `_count` metric.
 * The total count of all the `bucket_path` count values must be less than or equal to `indicator.doc_count`.
 * When utilizing this function, an initial calculation to gather the required `indicator` values is required.
-
+--
++
 .Properties of `count_correlation`
 [%collapsible%open]
 =====
 `indicator`:::
 (Required, object)
 The indicator with which to correlate the configured `bucket_path` values.
-
++
 .Properties of `indicator`
 [%collapsible%open]
-=====
+======
```

Comment:
This is indeed correct. Could you fix the below syntax example as well?

Correct
```
{
  "bucket_correlation": {
    "buckets_path": "range_values>_count", <1>
    "function": {
      "count_correlation": { <2>
        "indicator": {
          "expectations": [...],
          "doc_count": 10000
        }
      }
    }
  }
}```

Reply:
Good catch, I've updated in 0bdec771042d8f42a1f5e1b2b5a022d2672fdf86